src/components/TableRow.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import moment from "moment";
+//import CustomerProfile from "./CustomerProfile";
+
+function TableRow(props) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  function handleClick(event) {
+    event.preventDefault();
+    props.handleClick(props.data.id);
+  }
+  return (
+    <tr
+      onClick={() => setIsClicked(!isClicked)}
+      style={isClicked ? { backgroundColor: "yellow" } : null}
+    >
+      <th scope="row" />
+      <td>{props.data.id}</td>
+      <td>{props.data.title}</td>
+      <td>{props.data.firstName}</td>
+      <td>{props.data.surname}</td>
+      <td>{props.data.email}</td>
+      <td>{props.data.roomId}</td>
+      <td>{props.data.checkInDate}</td>
+      <td>{props.data.checkOutDate}</td>
```

Comment:
Destructured props would improve the readability here as you have to repeat `props.` many times